bin/usage.txt

```diff
@@ -0,0 +1,11 @@
+Usage: node ./bin/scss-to-json [options] [<file>]
```

Comment:
This should probably just be `Usage: node scss-to-json [options] [<file>]` and assume that people have it installed globally or know where the file is. Nobody will be running this command with `./bin/scss-to-json` though, which is why I don't think this makes sense.